misk-actions/src/main/kotlin/misk/web/interceptors/LogRequestResponse.kt

```diff
@@ -12,5 +12,10 @@ package misk.web.interceptors
  * methods of these objects will redact it.
  */
 @Retention(AnnotationRetention.RUNTIME)
-@Target(AnnotationTarget.FUNCTION)
-annotation class LogRequestResponse(val sampling: Double, val includeBody: Boolean)
+@Target(AnnotationTarget.FUNCTION, AnnotationTarget.CLASS)
+annotation class LogRequestResponse(
+  val rateLimiting: Long = 0,
```

Comment:
documentation? and I believe we agreed that `rateLimiting` should be nonzero? I think 10 qps is probably ok

Reply:
Ah yeah we did, updating

Reply:
Added documentation